build.assets/Dockerfile-centos7-fips

```diff
@@ -27,6 +27,54 @@ RUN mkdir -p /opt && cd /opt && \
     cd /opt/libbpf-${LIBBPF_VERSION}/src && \
     scl enable devtoolset-11 "make && BUILD_STATIC_ONLY=y DESTDIR=/opt/libbpf make install"
 
+
+
+FROM centos:7 AS boringssl
+# The below tools are required in order to build and compile the module:
+# Clang compiler version 7.0.1
+# Go programming language version 1.12.7
+# Ninja build system version 1.9.0
+#
+# We also need the FIPS 140-2 validated release of BoringSSL: ae223d6138807a13006342edfeef32e813246b39
+# For more information please refer to the section 12. Guidance and Secure Operation of:
+# https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3678.pdf
+
+# Install required dependencies.
+RUN yum groupinstall -y 'Development Tools' && \
+    yum install -y epel-release && \
+    yum update -y && \
+    yum -y install centos-release-scl-rh && \
+    yum install -y \
+    cmake3 \
+    llvm-toolset-7.0-clang-7.0.1 \
+    git
+
+
+RUN mkdir -p /opt && cd /opt && curl https://storage.googleapis.com/golang/1.12.7.linux-amd64.tar.gz | tar xz && \
```

Comment:
Do we even need go here? I know that the NIST guideline requires Go 1.12.7, but if we're not using it here, I don't think we need to install it.